imitation/trainer.py

```diff
@@ -59,6 +60,8 @@ def __init__(self, env, gen_policy, discrim, expert_policies, *,
 
     self.env = util.maybe_load_env(env, vectorize=True)
     self.gen_policy = gen_policy
+    if not isinstance(expert_policies, collections.abc.Sequence):
```

Comment:
Not a fan of multiple expert policies being an option in the first place. But we're soon switching to an API where this takes demos, so can probably ignore this for now.

Reply:
Agreed, that we shouldn't have multiple expert policies. This change was a crash-fix for the PR.